config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java

```diff
@@ -355,7 +355,43 @@ public void shouldLoadForksIgnoreClassicForkBlock() throws IOException {
                         StandardCharsets.UTF_8)));
     final GenesisConfigFile config = fromConfig(configNode);
 
-    assertThat(config.getForks()).containsExactly(1L, 2L, 3L, 3L, 1035301L);
+    assertThat(config.getForks()).containsExactly(1L, 2L, 3L, 1035301L);
```

Comment:
so duplicates weren't removed before? Or did it happen after this point?

Reply:
It happens later at ForkIdManager.java:58

These tests were expecting a duplicate value for `eip155Block` and `eip158Block` (see JsonGenesisConfigOptions.java:334), which the new logic doesn't do.

Reply:
We need to make sure this doesn't interfere with the legacy forkid issue with Besu 1.4 - @abdelhamidbakhta